linux/ibus-keyman/tests/scripts/run-tests.sh

```diff
@@ -97,23 +97,23 @@ function run_tests() {
     TMPFILE=$(mktemp)
     # mutter-Message: 18:56:15.422: Using Wayland display name 'wayland-1'
     mutter --wayland --headless --no-x11 --virtual-monitor 1024x768 &> "$TMPFILE" &
-    echo "kill -9 $!" >> "$PID_FILE"
+    echo "kill -9 $! || true" >> "$PID_FILE"
```

Comment:
Are these safe to proceed if the `kill -9` fails?

Reply:
Yes. It's just a cleanup task, so if `kill` fails it means that this one process keeps hanging around. But if we don't proceed then there will be more processes hanging around, so it's better to continue.